components/input/TextArea.tsx

```diff
@@ -104,6 +104,7 @@ const TextArea = forwardRef<TextAreaRef, TextAreaProps>(
           countWrapper: classNames(
             `${prefixCls}-textarea`,
             `${prefixCls}-textarea-show-count`,
+            { [`${prefixCls}-textarea-show-count-rtl`]: direction === 'rtl' },
```

Comment:
```suggestion
            `${prefixCls}-textarea-show-count-rtl`: direction === 'rtl',
```
How about this?

Reply:
We can't do that, we need the object to be able to use our direction condition